demo/midgard-sdk/src/tx-builder/user-events/deposit.ts

```diff
@@ -83,8 +83,10 @@ export const MintRedeemer = MintRedeemerSchema as unknown as MintRedeemer;
 export type FetchConfig = {
   depositAddress: Address;
   depositPolicyId: PolicyId;
-  inclusionStartTime: POSIXTime;
-  inclusionEndTime: POSIXTime;
+  inclusionTime?: {
+    start: POSIXTime;
+    end: POSIXTime;
+  }
```

Comment:
If we allow both to be optional, we'll be able to consider `undefined` as infinity.
```suggestion
  inclusionTimeLowerBound?: POSIXTime;
  inclusionTimeUpperBound?: POSIXTime;
}
```